HackYourFuture-CPH/chattie
# Fix add group

# Description
work on fixing adding a new room, need to add all users to channels_members, now only one user. Also, work on adding the current user as channel members to the newly created room. Add code from this task 'Image component added to RoomForm #301', I talk with Aswini about that. Because I change many files, she worked with the previous version. Add imageUrl to createchannel in channel-controller. When the user uploads the image, we create a new channel with the title, image URL, and send all data to the server. 

Fixes # (issue)

# How to test?
Click to create a new room button, add people, upload images and write the name of the group, write a message, you will see your message, and the new group in the list of the last conversation on the overview page.


# Checklist
